liquid-rust-core/src/resolve.rs

```diff
@@ -59,6 +61,66 @@ impl<'tcx> Resolver<'tcx> {
         Ok(Resolver { tcx, diagnostics, name_res_table, parent })
     }
 
+    /// `desugar_def_sig(sig)` translates a `DefFnSig` into a `core::FnSig`, and is used by
```

Comment:
TODO

Some of the functions here are weirdly suspicious and short. Probably they don't belong here either. They most likely will find their natural place once we remove this module.